app/tests/e2e/test_tenants.py

```diff
@@ -271,38 +271,9 @@ async def test_update_tenant_verifier_to_issuer(
     new_tenant = response.json()
     new_actor = await trust_registry.actor_by_id(tenant_id)
 
-    endorser_did = await acapy_wallet.get_public_did(governance_acapy_client)
-
-    async with get_tenant_controller(Role.TENANT, acapy_token) as tenant_controller:
-        public_did = await acapy_wallet.get_public_did(tenant_controller)
-
-        _connections = (await tenant_controller.connection.get_connections()).results
-
-        connections = [
-            connection
-            for connection in _connections
-            if connection.their_public_did == endorser_did.did
-        ]
-
-    endorser_connection = connections[0]
-
-    async with tenant_client(token=tenant["access_token"]) as client:
-        # Wait for connection to be completed
-        assert check_webhook_state(
-            client,
-            "connections",
-            {
-                "state": "completed",
-                "connection_id": endorser_connection.connection_id,
-            },
-        )
-
-    # Connection invitation
-    assert_that(endorser_connection).has_their_public_did(endorser_did.did)
```

Comment:
Why did you remove this part of the test? It checks whether the endorser connection uses the public did to connect

Reply:
ah darn - forgot to add that back in. 
Now getting token invalid error for the subwallet after updating. Am on it.